src/web/components/Onwards/ExactlyFive.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
```

Comment:
I think adding a story for this would be great, to highlight how the 5 cards would be rendered